package.json

```diff
@@ -18,7 +18,7 @@
           "dependencies": [
             "vite"
           ],
-          "pinVersion": ">=5 <8"
+          "pinVersion": "^7"
```

Comment:
I forced this version to 7 otherwise TypeScript throws during the compilation of the Vite plugins.

Reply:
In this case no, but this is forcing TypeScript to work correctly during the build.